.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Build/meta-mistysom"]
 	path = Build/meta-mistysom
 	url = https://github.com/MistySOM/meta-mistysom.git
+[submodule "Build/meta-mistylwb5p"]
```

Comment:
mind thart the fork is out of date, probably should/could be updated.

Reply:
When reading the difference, I can see that it only affects the bluetooth on lwb5p and not WiFi.